src/Symfony/Component/HttpClient/Tests/HttpClientTraitTest.php

```diff
@@ -120,6 +120,13 @@ public function provideResolveUrl(): array
         ];
     }
 
+    public function testResolveUrlWithoutScheme(): void
+    {
+        $this->expectException('Symfony\Component\HttpClient\Exception\InvalidArgumentException');
```

Comment:
`InvalidArgumentException::class`

Reply:
Other tests in this class the the string notation, should i update those as well?